### PR TITLE
Fix security alerts

### PR DESCRIPTION
**Pull request by cht8687**

### CODE REVIEW

**Comment by cht8687** [comment]
@stevemao seems travis node 6 build failed. Are we still supporting node 6?

**Comment by stevemao** [comment]
We can remove it

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   },
   "devDependencies": {
     "coveralls": "^2.11.9",
-    "eslint": "^3.0.0",
+    "eslint": ">=4.18.2",
```
Comment (stevemao) [inline package.json]: this should be ^4.18.2

Comment (cht8687) [reply package.json]: @stevemao  changed

**Pull request merged**